src/languages/sqf.js

```diff
@@ -2099,14 +2222,13 @@ export default function(hljs) {
     'setVehicleArmor',
     'setVehicleCargo',
     'setVehicleId',
-    'setVehicleInit',
     'setVehicleLock',
     'setVehiclePosition',
     'setVehicleRadar',
     'setVehicleReceiveRemoteTargets',
     'setVehicleReportOwnPosition',
     'setVehicleReportRemoteTargets',
-    'setVehicleTIPars',
+    'setVehicleTiPars',
```

Comment:
```suggestion
    'setVehicleTIPars',
```
https://community.bistudio.com/wiki/setVehicleTIPars